tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -101,7 +101,7 @@ struct dispatch_constants {
             prefetch_q_entries_ = 128;
             max_prefetch_command_size_ = 32 * 1024;
             cmddat_q_size_ = 64 * 1024;
-            scratch_db_size_ = 20 * 1024;
+            scratch_db_size_ = 19 * 1024;
```

Comment:
@aliuTT is #11316 still needed if this is getting reduced here?

Reply:
shouldn't be, I dropped this a fair amount.
also, when the memmap gets a little further along, that will free up a lot of space on idle eth for dispatch